frontend/javascripts/oxalis/model/accessors/flycam_accessor.ts

```diff
@@ -281,11 +281,13 @@ export function hasAdditionalCoordinates(
 
 export function getAdditionalCoordinatesAsString(
   additionalCoordinates: AdditionalCoordinate[] | null | undefined,
+  separator: string = ";",
 ): string {
   if (additionalCoordinates != null && additionalCoordinates.length > 0) {
     return additionalCoordinates
       ?.map((coordinate) => `${coordinate.name}=${coordinate.value}`)
-      .reduce((a: string, b: string) => a.concat(b, ";"), "") as string;
+      .reduce((a: string, b: string) => a.concat(b, separator), "")
+      .slice(0, -1 * separator.length);
```

Comment:
```suggestion
      .join(separator);
```

probably does what you want, if I understand the code correctly.

Reply:
oh nice, thanks!